policy/modules/services/ssh.te

```diff
@@ -174,6 +181,13 @@ userdom_use_user_terminals(ssh_t)
 # needs to read krb tgt
 userdom_read_user_tmp_files(ssh_t)
 
+# ssh re-executes itself during ProxyJump
+ssh_exec(ssh_t)
+
+tunable_policy(`ssh_connect_any_port',`
+	corenet_tcp_connect_generic_port(ssh_t)
```

Comment:
This doesn't seem to match the documented intent of the tunable above. I'd expect it to call one of the interfaces that have `all` in the name.